libr/core/linux_heap_glibc.c

```diff
@@ -968,6 +1043,30 @@ static void GH(print_heap_segment)(RCore *core, GH(RHeap_MallocState) *main_aren
 			}
 		}
 
+		#if __GLIBC_MINOR__ > 25
+		GH(RHeapTcache) *tcache = R_NEW0 (GH(RHeapTcache));
+		(void)r_io_read_at (core->io, brk_start + MALLOC_ALIGNMENT, (ut8 *)tcache, sizeof ( GH(RHeapTcache) ));
+		for(int i=0; i<TCACHE_MAX_BINS; i++){
+			if(tcache->counts[i]>0x0){
+				if ( (ut64)tcache->entries[i]-MALLOC_ALIGNMENT == (ut64)prev_chunk ){
+					is_free = true;
+					break;
+				}
+				if(tcache->counts[i]>0x1){
+					tcache_fd = (ut64)tcache->entries[i];
+					for(int n=1;n < tcache->counts[i];n++){
+						(void)r_io_read_at (core->io, tcache_fd, &tcache_tmp, sizeof ( GHT ) );
+						if ( (ut64)tcache_tmp-MALLOC_ALIGNMENT == (ut64)prev_chunk ) {
+							is_free = true;
+							break;
+						}
+						tcache_fd = (ut64)tcache_tmp;
+					}
+				}
+			}
+		}
+		#endif
```

Comment:
all preprocessor lines should start at the begining of the line, don't indent them